lib/prelude.dx

```diff
@@ -805,12 +851,73 @@ Dex's PRNG system is modelled directly after [JAX's](https://github.com/google/j
 '### Key functions
 
 -- TODO: newtype
-Key = Int64
-
-def hash (x:Key) (y:Int32) : Key = unsafeIO do
-  y64 = IToI64 y
-  %ffi threefry2x32 Int64 x y64
-def newKey (x:Int) : Key = hash (IToI64 0) x
+Key = Word64
+
+def threeFry2x32C (k:Int64) (count:Int64) : Int64 = unsafeIO do
```

Comment:
keeping around this implementation for comparison?

Reply:
I think it's fine to delete!

Reply:
Yes, please remove that one as well!

Reply:
Done.